src/theme.ts

```diff
@@ -22,7 +22,15 @@ const fonts = {
     'Apple Color Emoji',
     'Segoe UI Emoji',
   ]),
-  mono: fontStack(['SFMono-Regular', 'Consolas', 'Liberation Mono', 'Menlo', 'Courier', 'monospace']),
+  mono: fontStack([
+    'var(--fontStack-monospace)',
+    'SFMono-Regular',
+    'Consolas',
+    'Liberation Mono',
+    'Menlo',
+    'Courier',
+    'monospace',
+  ]),
```

Comment:
```suggestion
  mono: fontStack([
    'var(--fontStack-monospace,
    SFMono-Regular,
    Consolas,
    Liberation Mono,
    Menlo,
    Courier,
    monospace)',
  ]),
```

I _think_ it should be like this, but I'm not 100% sure.

Reply:
I am now 95% sure I am wrong.